File: Java/src/Boolean.java
import java.util.concurrent.ForkJoinPool;

public class Boolean {
    public static void main(String[] args) {
        System.out.println(a(-1,2));
        int threadCount = ForkJoinPool.getCommonPoolParallelism();
        System.out.println("Number of threads: " + threadCount);
    }

    public static boolean a(int i, int j) {
//        var sum = i + j;
//        return sum == 1 || (i == 1 || j == 1);
        return i == 1 || j == 1 || (i + j) == 1;
    }
}

File: Java/kotlin/Boolean.kt
package kotlin

import java.util.concurrent.ForkJoinPool

object Boolean {
    @JvmStatic
    fun main(args: Array<String>) {
        println(a(-1, 2))
        val threadCount = ForkJoinPool.getCommonPoolParallelism()
        println("Number of threads: $threadCount")
    }

    fun a(i: Int, j: Int): kotlin.Boolean {
        return i == 1 || j == 1 || (i + j) == 1
    }
}

